tools/utils/policy_test

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3
+import os
+import argparse
+import json
+import yaml
+import c7n.tests.common
+import c7n.filters.core
+import c7n.testing
+import c7n.utils
+import c7n.output
+
+@c7n.output.log_outputs.register('default')
+class NullOutput(object):
```

Comment:
The `LogOutput` class prints out the full stack trace when an exception is thrown. I'd like to catch this and print a friendly message when `Placebo` tries to access response file that doesn't exist:
```
OSError: response file (test/iam-group-test/iam.SimulatePrincipalPolicy_1.[json|pickle]) not found
```
Should I move this `NullOutput` into `c7n.output` or keep it here?

Reply:
having another pill subclass for this purpose might be preferred, I don't see that we want to alter the output class to silently ignore exceptions, versus this usage being explicit about handling the exceptions it wants to prettify. alternative register a test output subclass thats explicitly catching/checking the things it nows about.

Reply:
The latter option seems best. What's the correct way to register the new `Output` and to prevent the `default` one from firing for my test?